runtime/queries/column_numeric_histogram.go

```diff
@@ -124,19 +145,33 @@ func (q *ColumnNumericHistogram) Resolve(ctx context.Context, rt *runtime.Runtim
             -- fill in the case where we've filtered out the highest value and need to recompute it, otherwise use count.
             CASE WHEN high = (SELECT max(high) from histogram_stage) THEN count + (select c from right_edge) ELSE count END AS count
             FROM histogram_stage
-	      `, selectColumn, sanitizedColumnName, q.TableName, bucketSize)
+	      `,
+		selectColumn,
+		sanitizedColumnName,
+		q.TableName,
+		bucketSize,
+	)
+
 	histogramRows, err := rt.Execute(ctx, instanceID, priority, histogramSql)
 	if err != nil {
 		return err
 	}
 	defer histogramRows.Close()
+
 	histogramBins := make([]*runtimev1.NumericHistogramBins_Bin, 0)
 	for histogramRows.Next() {
+		var low, high sql.NullFloat64
 		bin := &runtimev1.NumericHistogramBins_Bin{}
-		err = histogramRows.Scan(&bin.Bucket, &bin.Low, &bin.High, &bin.Count)
+		err = histogramRows.Scan(&bin.Bucket, &low, &high, &bin.Count)
 		if err != nil {
 			return err
 		}
+		if !low.Valid || !high.Valid {
+			break
+		} else {
```

Comment:
Previously it would return all buckets, even empty buckets. Should we retain that property for now? Might be needed for chart rendering in frontend – and I think maybe it's normal to include the empty bins in histogram data? (Maybe we could make low/high be `google.protobuf.Value` type if setting them to `0` feels wrong?)

Reply:
Actually it's not possible for these values to be nulls we can revert the code back as it was.